setup.py

```diff
@@ -0,0 +1,15 @@
+from setuptools import setup, find_packages
```

Comment:
This probably belongs in a separate PR, as it is the initiating steps to pip install. OK to have here though (just link to this PR when making the official PIP implementation, and maybe change name of this PR to include setup.py)

Reply:
Ahhh, This was not my attempt to make `cytopipe` pip installable. In fact, the entry points (how a user communicates with `cytopipe`) must be explicitly written in a `setup.py` file. [source](https://setuptools.pypa.io/en/latest/userguide/entry_point.html#console-scripts)

But yes, in the future, making `cytopipe` pip installable will be a separate PR!

Here's the code block: https://github.com/WayScience/CytoPipe/pull/15/files/d4ef1fdf05febe35c7f8a5c9703a210bb5066d50#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7